string command_type
string time
string state
---
bool result
